src/main/resources/db.scripts.mvp/delete_duplicate_booking.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+    SET SCHEMA 'public';
+
+    CREATE TEMP TABLE tmp_visit_ids_to_be_deleted(
+        visit_id         int
+    );
+
+    --selection criteria for visits to be deleted
+    INSERT INTO tmp_visit_ids_to_be_deleted (visit_id) (
+        select v.id from visit v
+                join legacy_data ld on ld.visit_id = v.id
```

Comment:
Not a legacy visit I presume.

Reply:
I thought that but It was a legacy visit

Reply:
oh ok.. Will approve the change then